frontend/README.md

```diff
@@ -19,6 +19,10 @@ You will need the following things properly installed on your computer.
 
 ## Installation
 
+##Video Installation 
```

Comment:
Correct the Markdown and use `### Video Installation`.

Reply:
Fixed